docker-compose-mysql.yaml

```diff
@@ -1,56 +0,0 @@
-# Local development demo/example compose file
-# Composes:
-# - a MySQL database,
-# - a mlmd-server connected to the MySQL database,
-# - a model-registry server connected to the mlmd-server.
-version: '3'
-services:
-  mlmd-server:
-    image: gcr.io/tfx-oss-public/ml_metadata_store_server:1.14.0
-    container_name: mlmd-server
-    depends_on:
-      mysql:
-        condition: service_healthy
-        required: true
-    ports:
-      - "9090:9090"
-    entrypoint: ["/bin/metadata_store_server"]
-    command: [
-      "--grpc_port=9090",
-      "--mysql_config_database=model_registry",
-      "--mysql_config_host=mysql",
-      "--mysql_config_port=3306",
-      "--mysql_config_user=root",
-      "--mysql_config_password=demo",
-      "--enable_database_upgrade=true"
-    ]
-  model-registry:
-    image: ghcr.io/kubeflow/model-registry/server:latest
```

Comment:
I would still keep this `docker-compose-mysql.yaml`, it's meant to use the published image without requiring a local rebuild of the docker container, wdyt ?

Reply:
sorry didn't notice `docker-compose.yaml` takes the "role" of what-was `docker-compose-mysql.yaml`

Reply:
no worries 😁